FAIL
Kernel panic in run: PANIC at ../../threads/thread.c:305 in thread_current(): assertion `is_thread (t)' failed.
Call stack: 0x8004217ff9 0x800420715e 0x800420a5d0 0x800420a4e3 0x8004213756 0x800421ff46 0x800421f4e5 0x800421f5b5 0x800421df2b 0x800421c082 0x800421bb8f 0x800421d318 0x800421cf18 0x800421cd95
Translation of call stack:
0x0000008004217ff9: debug_panic (lib/kernel/debug.c:32)
0x000000800420715e: thread_current (threads/thread.c:306)
0x000000800420a5d0: lock_held_by_current_thread (threads/synch.c:256)
0x000000800420a4e3: lock_release (threads/synch.c:240)
0x0000008004213756: disk_read (devices/disk.c:227)
0x000000800421ff46: inode_read_at (filesys/inode.c:215)
0x000000800421f4e5: lookup (filesys/directory.c:88 (discriminator 1))
0x000000800421f5b5: dir_lookup (filesys/directory.c:112)
0x000000800421df2b: filesys_open (filesys/filesys.c:87)
0x000000800421c082: load (userprog/process.c:475)
0x000000800421bb8f: process_exec (userprog/process.c:230)
0x000000800421d318: sys_exec (userprog/syscall.c:192)
0x000000800421cf18: syscall_handler (userprog/syscall.c:85)
0x000000800421cd95: no_sti (userprog/syscall-entry.o:?)
